Documentation/api.md

```diff
@@ -1151,6 +1151,7 @@ ThanosSpec defines parameters for a Prometheus server within a Thanos deployment
 | grpcServerTlsConfig | GRPCServerTLSConfig configures the gRPC server from which Thanos Querier reads recorded rule data. Note: Currently only the CAFile, CertFile, and KeyFile fields are supported. Maps to the '--grpc-server-tls-*' CLI args. | *[TLSConfig](#tlsconfig) | false |
 | logLevel | LogLevel for Thanos sidecar to be configured with. | string | false |
 | logFormat | LogFormat for Thanos sidecar to be configured with. | string | false |
+| LoggingConfig | LoggingConfig configures request logging in Thanos. | *[v1.SecretKeySelector](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/#secretkeyselector-v1-core) | false |
```

Comment:
why this is starting from an uppercase and not lowercase like other fields? :thinking:

Reply:
Simon points it out below, I had the tag wrong.